setup.py

```diff
@@ -593,17 +633,21 @@ def get_common_options(build_ext):
         raise DistutilsError('HOROVOD_GPU_BROADCAST=%s is invalid, supported '
                              'values are "", "MPI".' % gpu_broadcast)
 
+    have_cuda = False
+    have_rocm = False
+    cuda_include_dirs = cuda_lib_dirs = cuda_macros = []
     if gpu_allreduce or gpu_allgather or gpu_broadcast:
-        have_cuda = True
-        cuda_include_dirs, cuda_lib_dirs = get_cuda_dirs(build_ext, cpp_flags)
-    else:
-        have_cuda = False
-        cuda_include_dirs = cuda_lib_dirs = []
+        if os.environ.get('HOROVOD_GPU_ROCM'):
```

Comment:
Instead of having the user specify `HOROVOD_GPU_ROCM`, how about we use the existing `HOROVOD_GPU_ALLREDUCE` env variable and have them set `HOROVOD_GPU_ALLREDUCE=RCCL`?

Reply:
I'm typically in favor of simplifying, but in this case, shouldn't we also support building with ROCm support but without RCCL, i.e. ROCm with RDMA but without RCCL?